host_applications/linux/apps/raspicam/RaspiStill.c

```diff
@@ -826,6 +826,18 @@ static MMAL_STATUS_T create_camera_component(RASPISTILL_STATE *state)
    video_port = camera->output[MMAL_CAMERA_VIDEO_PORT];
    still_port = camera->output[MMAL_CAMERA_CAPTURE_PORT];
 
+   {
```

Comment:
I'm not totally sure what this does or if it's necessary - it came from 6by9's original snippet.
In fact, it kind of looks broken to me since I don't see what block the curly braces are supposed to go with. 
Also, it kind of looks like a very similar block exists on [line 1051 of RaspiCamControl.c](https://github.com/OsmoSystems/userland/blob/master/host_applications/linux/apps/raspicam/RaspiCamControl.c#L1051). The snippet is from 3 years ago, so maybe that block didn't used to exist?

**Option 0:** I could just leave it, since it doesn't _seem_ to hurt anything.
**Option 1:** I could try taking it out and see if the LED control still works. Slightly annoying since it takes a little while to build & test.

Reply:
Option 0 seems fine to me (with comments added mentioning what you just said) for the sake of not losing any more time to this

Reply:
[major] I vote for option 1...

Reply:
I'm fine with either 0 or 1, it doesn't seem like it will make a difference either way.

Reply:
welp, I went ahead and did Option 1. When I removed this block, the patch no longer worked! No idea why, but I guess I'll leave it in, and I'll add a comment.

Reply:
this may explain why the picamera version we attempted, didn't work - this stuff must be subscribing to the appropriate events somehow.